.github/workflows/linux-build-and-test.yaml

```diff
@@ -101,31 +101,35 @@ jobs:
         file: ./build/codecov/timescaledb-codecov.info
 
     - name: Show regression diffs
-      if: failure()
+      if: always()
+      id: collectlogs
       run: |
         find . -name regression.diffs -exec cat {} + > regression.log
         find . -name postmaster.log -exec cat {} + > postgres.log
+        if [[ "${{ runner.os }}" == "Linux" ]] && coredumpctl list; then echo "::set-output name=coredumps::true"; fi
```

Comment:
Not sure what this line is about. AFAICT, this will show the list of coredumps  and if there are any, and in that case turn on the coredumps variable. Is it intentional to show the list of coredumps?

Reply:
was not intentional, i removed it